x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/entity_flyout.cy.ts

```diff
@@ -62,28 +62,25 @@ describe(
     },
   },
   () => {
-    before(() => {
+    beforeEach(() => {
```

Comment:
negligable performance impact here of running the archiver before every test. I only did this because of the custom delete code needed in "after", I thought simpler to move to this approach and combine before and beforeEach